docs/discover/document-data.asciidoc

```diff
@@ -1,69 +1,57 @@
 [[document-data]]
-== Viewing Document Data
+== Viewing document data
 
-When you submit a search query, the 500 most recent documents that match the query
-are listed in the Documents table. You can configure the number of documents shown
-in the table by setting the `discover:sampleSize` property in <<advanced-options,
-Advanced Settings>>. By default, the table shows the localized version of the time
-field configured for the selected <<index-patterns, index pattern>> and the document `_source`. You can
-<<adding-columns, add fields to the Documents table>> from the Fields list.
-You can <<sorting, sort the listed documents>> by any indexed field that's included
-in the table.
-
-To view a document's field data, click the *Expand* button 
-image:images/ExpandButton.jpg[Expand Button] to the left of the document's table
-entry. 
-
-image::images/Expanded-Document.png[]
-
-To view the original JSON document (pretty-printed), click the *JSON* tab.
-
-To view the document data as a separate page, click the *View single document* 
-link. You can bookmark and share this link to provide direct access to a 
-particular document.
-
-To display or hide a field's column in the Documents table, click the
-image:images/add-column-button.png[Add Column] *Toggle column in table* button.
-
-To collapse the document details, click the *Collapse* button 
-image:images/CollapseButton.jpg[Collapse Button].
+When you submit a search query in *Discover*, the most recent documents that match the query
+are listed in the documents table. 
+By default, the table includes columns for 
+the time field and the document _source, which shows all fields and values in the document.
 
 [float]
 [[sorting]]
-=== Sorting the Document List
-You can sort the documents in the Documents table by the values in any indexed
-field. If a time field is configured for the current index pattern, the 
-documents are sorted in reverse chronological order by default.
-
-To change the sort order, hover over the name of the field you want to sort by
-and click the sort button. Click again to reverse the sort order.
+=== Add and remove fields from the document table
+
+Use the following commands to 
+tailor the documents table to show specific events.
+
+Click the 
+arrow to expand the row to view the details in table or JSON format.
+
+[horizontal]
+Add a field column:: 
+Hover over the list of *Available fields* and then click *add* next to each field you want include in the table.
+The first column you add replaces the _source column.  
+Change sort order:: By default, columns are sorted by the values in the field.  
+If a time field is configured for the current index pattern, 
+the documents are sorted in reverse chronological order.  To change the sort order, hover over the column 
+and click image:images/sort-icon.png[] 
+Move a field column:: Hover over the column header and click the move left (<<) or move right icon (>>). 
```

Comment:
He we should add that you can sort by multiple values. So the first click on a fields defines the primary sort field.
When you click on another field, the secondary field is added, and so on ...
when you click on an additional field 
- the first click -  it's sorted ascending, 
- the second click - it's sorted descending, 
- the 3rd klick - removes the field from sorted fields